<?xml version="1.0" encoding="UTF-8"?>

<launch>
  <include file ="$(find ublox_gps)/launch/ublox_device.launch"/>
  <node pkg="ntrip_ros" type="ntripclient.py" name="ntrip_ros" output="screen">
 	 <param name="rtcm_topic" value="/ublox_gps/rtcm"/>  //rtcm 토픽
 	 <param name="ntrip_server" value="RTS1.ngii.go.kr:2101"/> //접속경로
 	 <param name="ntrip_user" value="ICASjuni1"/> //아이디
 	 <param name="ntrip_pass" value="ngii"/> //비번
 	 <param name="ntrip_stream" value="RTK-RTCM32"/> //마운트 포인트
 	 <param name="nmea_gga" value="$GPGGA,113937.335,3549.931,N,12845.211,E,1,12,1.0,0.0,M,0.0,M,,*6D"/>
  </node>
</launch>

